src/test/java/com/algorand/algosdk/unit/AlgodPaths.java

```diff
@@ -151,4 +151,19 @@ public void getBlockTimestampOffset() {
     public void setBlockTimestampOffset(Long round) {
         ps.q = algodClient.SetBlockTimeStampOffset(round);
     }
+
+    @When("we make a GetLedgerStateDelta call against round {int}")
```

Comment:
It looks like you can also use `Long` instead of `int`:

```suggestion
    @When("we make a GetLedgerStateDelta call against round {long}")
        public void we_make_a_get_ledger_state_delta_call_against_round(Long round) {
```